scalafmt-core/shared/src/main/scala/org/scalafmt/util/FileOps.scala

```diff
@@ -40,12 +41,16 @@ object FileOps {
     */
   def readFile(filename: String)(implicit codec: Codec): String = {
     if (filename matches "https?://.*") {
-      scala.io.Source.fromURL(filename)("UTF-8").getLines().mkString("\n")
+      readURL(new URL(filename))
```

Comment:
Either we can delegate exception to `Source.fromURL` or we can validate this URL properly.
This API is internal, as I understood. CLI option is validated properly.

Reply:
This is fine.